src/py42/_internal/clients/alertrules/exfiltration.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from py42.clients import BaseClient
+
+
+class Exfiltration(BaseClient):
+
+    _version = u"v1"
+    _resource = u"query-endpoint-exfiltration-rule"
+    _api_prefix = u"/svc/api/{0}/Rules/{1}".format(_version, _resource)
+
+    def __init__(self, session, tenant_id):
```

Comment:
Instead of passing `user_context`, passed `tenant_id` , a little optimisation since `get_current_tenant_id`  is the only method in `UserContext`, I felt passing it is unnecessary and it makes a network call each time it is called. Is this ok?

Did this change only in few places considering may be I have to revert it back if this is incorrect.

Reply:
makes sense to me!